src/codec_svt.c

```diff
@@ -128,13 +128,15 @@ static avifResult svtCodecEncodeImage(avifCodec * codec,
         svt_config->rc_twopass_stats_in = (SvtAv1FixedBuf) { NULL, 0 };
 #endif
 
+        svt_config->rate_control_mode = 0; // CRF because enable_adaptive_quantization is 2
```

Comment:
Line 122 and line 131 are actually the defaults, so I can also delete these two lines. Would you prefer that?

Reply:
I think it’s best to keep them in case the default changes